alma/controllers/hook/GetContentHookController.php

```diff
@@ -101,6 +102,7 @@ public function processConfiguration()
                     $almaPlans[$key]['enabled'] = 1;
                     $almaPlans[$key]['min'] = $feePlan->min_purchase_amount;
                     $almaPlans[$key]['max'] = $feePlan->max_purchase_amount;
+                    $almaPlans[$key]['deferred'] = $feePlan->deferred_trigger_limit_days;
```

Comment:
Even if this works magically, `deferred_trigger_limit_days` property have to be explicitly added into alma-php-client `FeePlan` entity

Reply:
the config key `deferred` is not sufficiently eloquent and can be confused with `isDeferred()`

Please keep the API name here to avoid confusion and name your config in the same way => `deferred_trigger_limit_days` or `deferredTriggerLimitDays`

further, `deffered` seems to be a boolean, not an integer 😉

Reply:
deferred_trigger_limit_days is null if empty or int value if filled.

Reply:
I rename deferred by deferred_trigger_limit_days :)